src/components/shared/transactionsTable/transactionsTable.css

```
@@ -0,0 +1,42 @@
+.address {
+  display: flex;
+  align-items: center;
+
+  & .addressValue {
+    margin-left: 10px;
+  }
+}
+
+.sortingColumn {
+  cursor: pointer;
+}
+
+.arrow {
+  width: 0;
+  height: 0;
+  margin-bottom: 5px;
+  margin-left: 5px;
+  border-left: 4px solid transparent;
+  border-right: 4px solid transparent;
+
+  &Up {
```

This is smart, I didn't know you can define a className like this.

But it makes it impossible succesfully to `ctrl-f` for `arrowUp` in this file, which is what I would do to see how this class is defined.

```suggestion
  &.arrowUp {
```

---

I didn't think of that. Changed!